frontend/src/pages/ForgotPasswordPage.tsx

```diff
@@ -0,0 +1,152 @@
+    disableSending();
+  };
+
+  const sendLinkView = () => {
+    return (
+      <div className="formDiv">
+        <ModalHeader>
+          <h2>
+            <Trans i18nKey="Forgot password" />
+          </h2>
+        </ModalHeader>
+        <ModalContent gap={30}>
+          <div className="formSubtitle">
+            <Trans i18nKey="Forgot password subtitle" />
+          </div>
+          <form onSubmit={handleSend}>
+            <Input
+              label={t('Your email')}
+              required
+              id="email"
+              type="email"
+              placeholder={t('Email')}
+              value={email}
+              onChange={(e) => setEmail(e.currentTarget.value)}
+            />
+            <button className="button-large" type="submit">
+              <Trans i18nKey="Send link" />
+            </button>
+          </form>
+          <div className="formFooter">
+            <Trans i18nKey="Remembered password" />{' '}
+            <Link to={`${paths.loginPage}`}>
+              <Trans i18nKey="Log in" />
+            </Link>
+          </div>
+        </ModalContent>
+      </div>
+    );
+  };
+
+  const linkSentView = () => {
+    return (
+      <div className="formDiv">
+        <ModalHeader>
+          <h2>
+            <Trans i18nKey="Reset link sent" />
+          </h2>
+        </ModalHeader>
+        <ModalContent gap={30}>
+          <div className={classes(css.secondSubtitle)}>
+            <MailIcon />
+            <p>
+              <Trans i18nKey="We have sent an email" values={{ email }} />
+            </p>
+          </div>
+          <div className="formFooter" style={{ textAlign: 'center' }}>
+            <div style={{ marginBottom: '16px' }}>
+              <Trans i18nKey="Wrong email address?" />{' '}
+              <button
+                className={classes(css.linkButton, css.green)}
+                type="button"
+                onClick={() => setView('Send link')}
+              >
+                <Trans i18nKey="Change address" />
+              </button>
+            </div>
+            <div>
+              {!resendDisabled && (
+                <>
+                  <Trans i18nKey="Didn’t receive the email" />{' '}
+                  <button
+                    type="button"
+                    className={classes(css.linkButton, css.green)}
+                    onClick={handleResend}
+                  >
+                    <Trans i18nKey="Resend link" />
+                  </button>
+                </>
+              )}
+              {resendDisabled && (
+                <>
+                  <Trans i18nKey="Reset link sent successfully" />{' '}
+                  <Trans i18nKey="Retry timer" values={{ time: disableTime }} />
+                </>
+              )}
+            </div>
+          </div>
+        </ModalContent>
+      </div>
+    );
+  };
+
+  return (
+    <>
+      {view === 'Send link' && sendLinkView()}
+      {view === 'Link sent' && linkSentView()}
```

Comment:
```suggestion
      {linkSent ? linkSentView() : sendLinkView()}
```

The `view` would make sense as a boolean (or typed enum e.g. `useState<'a' | 'b' | 'c'>('a')` if there is more than two options)